app/src/components/OrderBookSnapshot.tsx

```diff
@@ -116,7 +120,7 @@ class OrderBookSnapshot extends Component<WithStyles, State> {
      * @returns {number}
      */
     getNumDataPoints = (): number => {
-        return Math.trunc(window.screen.width * window.devicePixelRatio * 0.76);
+        return Math.trunc(window.screen.width * window.devicePixelRatio * 0.5);
```

Comment:
Since there is a Constants file, shouldn't we place this one there?

Reply:
You're right, I'll move it